dtos/address.go

```diff
@@ -70,12 +82,15 @@ func NewRESTAddress(host string, port int, httpMethod string) Address {
 
 type MQTTPubAddress struct {
 	Publisher      string `json:"publisher,omitempty" validate:"required"`
-	Topic          string `json:"topic,omitempty" validate:"required"`
 	QoS            int    `json:"qos,omitempty"`
 	KeepAlive      int    `json:"keepAlive,omitempty"`
 	Retained       bool   `json:"retained,omitempty"`
 	AutoReconnect  bool   `json:"autoReconnect,omitempty"`
 	ConnectTimeout int    `json:"connectTimeout,omitempty"`
+	Scheme         string `json:"scheme" validate:"required"`
+	SecretPath     string `json:"secretPath" validate:"required"`
+	AuthMode       string `json:"authMode" validate:"required,oneof='none' 'usernamepassword' 'cacert' 'clientcert'"`
+	SkipCertVerify bool   `json:"skipCertVerify"`
```

Comment:
omitempty

Reply:
Thanks for catch that. 
Added omitted to each fields because MQTTPubAddress is the part of Address DTO and we want to omit the fields for other struct like RESTAddress or EmailAddress in query result.